app/views/prices.jade

```diff
@@ -0,0 +1,296 @@
+    //- CLIENTS
+    .clients
+        .container
+            //- PRICING PLANS
+            #pricing.pricing-tables.pricing-tables-mobile.scrollspy
+                .plans-head.text-center
+                    h4.headTitles.noMargBot #{prices.rates.rate_intro}
+                    p #{prices.rates.rate_sub}
+                    .col-xs-12.col-sm-offset-3.col-sm-8.incluye
+                        ul
+                            li
+                                i.fa.fa-commenting-o
+                                | #{prices.rates.sub1}
+                            li
+                                i.fa.fa-shopping-cart
+                                | !{prices.rates.sub2}
+                            li
+                                i.fa.fa-tablet
+                                | #{prices.rates.sub3}
+                            li
+                                i.fa.fa-graduation-cap
+                                | #{prices.rates.sub4}
+                    .clearfix
+                .row
+                    .col.s12.offset-m2.m8
+                        ul.tabs
+                            li.tab.col.s12.m6
+                                a.active(href='#mensual') !{prices.rates.monthly}
+                            li.tab.col.s12.m6
+                                a(href='#anual') !{prices.rates.yearly}
+                    // Plan Mensual
+                    #mensual.col.s12
+                        .col-xs-12.col-md-3.mobilePrices
+                            .pricing-table-grid
+                                h3 #{prices.rates.basic}
+                                ul
+                                    li
+                                        span
+                                            | 9,95
+                                            em #{prices.rates.euro_month}
+                                    li
+                                        //- MIXIN Use
+                                        +webapp
+                                    li
+                                        a(href='#')
+                                            i.fa.fa-bar-chart
+                                            | !{prices.rates.statistics}
+                                    li
+                                        a(href='#')
+                                            i.fa.fa-bullhorn
+                                            | !{prices.rates.bullhorn}
+                                    li
+                                        a(href='#')
+                                            i.fa.fa-pencil
+                                            | !{prices.rates.sign}
+                                    li
+                                        a(href='#')
+                                            i.fa.fa-bell-slash
+                                            | !{prices.rates.no_push}
+                                a.order-btn.modal-trigger(href='#pruebaGratis') #{home.register.call_to_action}
```

Comment:
we must use home keys only in the home.jade and we only use prices keys only in prices.jade. If you want to reuse the same keys, you must use different namespace (like navbar or header or login)